crypto/rand/rand_win.c

```diff
@@ -567,51 +166,6 @@ int RAND_poll(void)
     return (1);
 }
 
-int RAND_event(UINT iMsg, WPARAM wParam, LPARAM lParam)
```

Comment:
I'm wondering whether we should retain RAND_event(), and just have it call RAND_poll() (ignoring all params), and return RAND_status()? i.e. for backwards compat.

Reply:
> Oh...which also means the documentation needs to change too :-)
I noted that too here:
https://github.com/openssl/openssl/blob/1dc1ea182be183d8a393fdce4494360aee059cd2/NOTES-Windows.txt#L167

This change means `gdi32.dll` is no longer needed?

Reply:
_Note: See also this thread on [openssl-users](https://mta.openssl.org/pipermail/openssl-users/2020-July/012800.html)._